assets/wix/bundle.wxs

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:bal="http://schemas.microsoft.com/wix/BalExtension">
+  <!-- UpgradeCode GUID MUST REMAIN SAME THROUGHOUT ALL VERSIONS, otherwise, updates won't occur. -->
+  <?if $(sys.BUILDARCH)=x64?>
+    <?define UpgradeCodePreview = "3C90221B-D500-43C6-A4A6-0BE6C2C1B317"?>
```

Comment:
These aren't the same as the MSI, are they? I'd make them different, but in the same matrix you have. The package ref-counting feature I wrote into MSIs and WiX does use these, so ref-counting could get off and prevent uninstalls if you reuse them.

Reply:
Looks like from the tests below (👏) these are different, but I'll leave the comment here as just an FYI.

Reply:
These are different than the MSI.

Reply:
I'm verifying the original MSI guid in the tests because I need the MSI detection to work.  The burn wrapper should be an implementation detail.  I can add tests for both if you think it is needed.